src/stores.js

```diff
@@ -4,248 +4,81 @@
 var DiscoveryFeedQuoteStore = new Lang.Class({
     Name: 'DiscoveryFeedQuoteStore',
-    Extends: GObject.Object,
-    Properties: {
-        quote: GObject.ParamSpec.string('quote',
-                                          '',
-                                          '',
-                                          GObject.ParamFlags.READWRITE |
-                                          GObject.ParamFlags.CONSTRUCT_ONLY,
-                                          ''),
-        author: GObject.ParamSpec.string('author',
-                                           '',
-                                           '',
-                                           GObject.ParamFlags.READWRITE |
-                                           GObject.ParamFlags.CONSTRUCT_ONLY,
-                                           '')
-    }
+    Extends: EosDiscoveryFeed.QuoteCardStore
 });
 
 // eslint-disable-next-line no-unused-vars
 var DiscoveryFeedWordQuotePairStore = new Lang.Class({
     Name: 'DiscoveryFeedQuotePairStore',
-    Extends: DiscoveryFeedCardStore,
-    Properties: {
-        quote: GObject.ParamSpec.object('quote',
-                                          '',
-                                          '',
-                                          GObject.ParamFlags.READWRITE |
-                                          GObject.ParamFlags.CONSTRUCT_ONLY,
-                                          DiscoveryFeedQuoteStore.$gtype),
-        word: GObject.ParamSpec.object('word',
-                                         '',
-                                         '',
-                                         GObject.ParamFlags.READWRITE |
-                                         GObject.ParamFlags.CONSTRUCT_ONLY,
-                                         DiscoveryFeedWordStore.$gtype)
-    },
-
-    _init: function(params) {
-        params.type = CARD_STORE_TYPE_WORD_QUOTE_CARD;
-        this.parent(params);
-    }
+    Extends: EosDiscoveryFeed.WordQuoteCardStore
 });
 
-var LAYOUT_DIRECTION_IMAGE_FIRST = 0;
-var LAYOUT_DIRECTION_IMAGE_LAST = 1;
+var LAYOUT_DIRECTION_IMAGE_FIRST = EosDiscoveryFeed.CardLayoutDirection.IMAGE_FIRST;
+var LAYOUT_DIRECTION_IMAGE_LAST = EosDiscoveryFeed.CardLayoutDirection.IMAGE_LAST;
 
-var THUMBNAIL_SIZE_APP_STORE = 180;
-var THUMBNAIL_WIDTH_APP_STORE_LINK = 211;  // eslint-disable-line no-unused-vars
-var THUMBNAIL_SIZE_ARTICLE = 254;
-var THUMBNAIL_SIZE_NEWS = 322;  // eslint-disable-line no-unused-vars
-var THUMBNAIL_SIZE_ARTWORK = 400;
+var THUMBNAIL_SIZE_APP_STORE = EosDiscoveryFeed.THUMBNAIL_SIZE_APP_STORE;
+var THUMBNAIL_WIDTH_APP_STORE_LINK = EosDiscoveryFeed.THUMBNAIL_WIDTH_APP_STORE_LINK;  // eslint-disable-line no-unused-vars
+var THUMBNAIL_SIZE_ARTICLE = EosDiscoveryFeed.THUMBNAIL_SIZE_ARTICLE;
+var THUMBNAIL_SIZE_NEWS = EosDiscoveryFeed.THUMBNAIL_SIZE_NEWS;  // eslint-disable-line no-unused-vars
+var THUMBNAIL_SIZE_ARTWORK = EosDiscoveryFeed.THUMBNAIL_SIZE_ARTWORK;
```

Comment:
Again, transitional - here to avoid churn.

Reply:
Same as above.